src/validator_config.rs

```diff
@@ -0,0 +1,45 @@
+use solana_sdk::pubkey::Pubkey;
+
+pub struct ValidatorConfig {
+    pub tpu_enable_udp: bool,
+    pub tpu_disable_quic: bool,
+    pub max_ledger_size: Option<u64>,
+    pub skip_poh_verify: bool,
+    pub no_snapshot_fetch: bool,
+    pub require_tower: bool,
+    pub enable_full_rpc: bool,
+    pub known_validators: Option<Vec<Pubkey>>,
```

Comment:
nit: The logic later of initializing it in `add_known_validator` is a bit confusing, could this just be a straight-up `Vec`?

Reply:
yes will fix. i think i got addicted to using Option<T> early on. Will try to avoid when not needed lol